### ISSUE TITLE
Automated rules started with "Cryostat" template only record on cryostat container

### ISSUE THREAD

**Issue by jan-law** [cryostatio/cryostat#933]
Not sure if this is a bug or not, but if you start an automated rule using the web UI and smoketest with MatchExpression `true` and `template.name: Cryostat`, I only see one recording started on the Cryostat container. Is this intentional?

**Comment by andrewazores** [comment]
That's "intentional". The template with name `Cryostat` is [bundled into the Cryostat OCI image](https://github.com/cryostatio/cryostat/blob/main/src/main/extras/cryostat.jfc), so other JVMs present in the smoketest setup don't have any template by that name and recording start requests initiated by that rule will fail in the background.

**Comment by jan-law** [comment]
Hm, maybe we could hide the Cryostat template on the automated rules template dropdown in case someone else tries to use it.

**Comment by andrewazores** [comment]
Good point. The template selection dropdown should probably update based on the selected target, rather than specifically removing the Cryostat template.

**Comment by andrewazores** [comment]
It appears I even foresaw this issue and then promptly forgot about it:

https://github.com/cryostatio/cryostat-web/blob/d4e92765e1238e15766427f6845824d37ef18dff/src/app/Rules/CreateRule.tsx#L157
